sled-agent/config-reconciler/src/reconciler_task/zones.rs

```diff
@@ -76,6 +87,16 @@ impl OmicronZones {
         Self { zones: IdMap::default(), mount_config, timesync_config }
     }
 
+    pub(crate) fn has_retryable_error(&self) -> bool {
+        self.zones.iter().any(|zone| match &zone.state {
+            ZoneState::Running(_) => false,
+            // Assume any error is retryable. This might not be right? We can
+            // narrow this down in the future.
+            ZoneState::PartiallyShutDown { .. }
```

Comment:
.... I think this answers my comment above, that "partially shut down zones" are retryable.